Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM backstopjs/backstopjs
+
+# Creates a Docker group and Jenkins user so that files created within the container are modifiable by the host.
+
+RUN addgroup -g 999 docker
+RUN adduser -Ds /bin/bash -u 106 -G docker jenkins
+USER jenkins
```

Comment:
What are the flags and numbers here?

Reply:
addgroup:
* `-g` sets the group ID to use when creating the group

adduser:
* `-D` means "Don't assign a password"
* `-s /bin/bash` sets the default shell for the user.
* `-u 106` sets the userid to use when creating the user
* `-G docker` adds the user to the 'docker' group.

See references here: https://busybox.net/downloads/BusyBox.html

Reply:
Seems legit. Wasn't sure about the Ds